addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -40,13 +56,23 @@
 // Style
 // ------------------------------------------------------------------
 
-.o_ChatterTopbar {
+.o_ChatterTopbar.o-active {
+    color: $o-brand-odoo;
+    background-color: lighten(gray('300'), 7%);
+    border-color: gray('300');
+    border-bottom-color: transparent;
+    border-top-color: transparent;
+}
+
+.o_ChatterTopbar_buttonClose {
+    border-radius: 0 0 10px 10px;
+    font-size: $font-size-lg;
+    background-color: gray('700');
+    color: gray('100');
+    cursor: pointer;
 
-    .o-active {
-        color: $o-brand-odoo;
-        background-color: lighten(gray('300'), 7%);
-        border-color: gray('300');
-        border-bottom-color: transparent;
-        border-top-color: transparent;
+    &:hover {
+        background-color: gray('600');
+        color: $white;
```

Comment:
change of color on hover is barely noticeable

Reply:
there were no color change before that commit so I wouldn't make it too visual. What should be improved ? More difference in the color or in the background-color ?

Reply:
nah you can keep like this, just noticed that